src/ostorlab/cli/scanner/scanner.py

```diff
@@ -54,32 +57,52 @@ def scanner(
     state_reporter = scanner_state_reporter.ScannerStateReporter(
         scanner_id=scanner_id, hostname=socket.gethostname(), ip=ip.get_ip()
     )
+    nb_parallel_scans = int(n)
```

Comment:
we need to check that nb_parallel_scans > 0

Reply:
It is already the case: `type=click.IntRange(1, None)` will make the sub-command fail if it is `< 0`.